lib/rumoji/emoji.rb

```diff
@@ -53,8 +57,14 @@ def multiple?
 
     ALL_REGEXP = Regexp.new(ALL.map(&:string).join('|'))
 
+    SYMBOL_LOOKUP = ALL.each.with_object({}) do |emoji, lookup|
+      emoji.symbols.each do |symbol|
+        lookup[symbol.to_s] = emoji
+      end
+    end
+
     def self.find(symbol)
-      ALL.find {|emoji| emoji.include? symbol }
+      SYMBOL_LOOKUP[symbol.to_s]
```

Comment:
No idea if it is worthwhile, but if we added both String and Symbol keys to `SYMBOL_LOOKUP` for each Emoji, we could avoid the creation of a new String here.